requirements.txt

```diff
@@ -1,5 +1,5 @@
-ops >= 1.5.0
-deepdiff
-lightkube
-lightkube-models
+ops
+deepdiff == 6.2.2
```

Comment:
Let's add a comment here to know when we should un-pin this version.

Reply:
+1

Reply:
Deepdiff isn't part of `requirements.txt` anymore.